server-sdk/src/main/java/com/amazonaws/ml/mms/servingsdk/Context.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.amazonaws.ml.mms.servingsdk;
+
+import java.util.Properties;
+
+/**
+ * This interface provides access to the current running Model Server.
+ */
+public interface Context {
+    /**
+     * Get the confuguration of the current running Model Server
+     * @return Properties
+     */
+    Properties getConfig();
+
+
+    //public Map<String, Model> getModels();
```

Comment:
Not sure what happen to my earlier comments, they seems gone.

Remove unused code

Reply:
Sure. I can remove this.